list-cookies/cookies.js

```diff
@@ -0,0 +1,13 @@
+window.onload = function(e){
+    //get all cookies in the domain
+    chrome.cookies.getAll({domain: "developer.mozilla.org"},  function(cookies) {
+        var cookieList = document.getElementById('cookie-list');
+        //add an <li> item with the name and value of the cookie to the list
+        for (x in cookies) {
```

Comment:
I'd rather see `for...of` than `for...in` here: https://hacks.mozilla.org/2015/04/es6-in-depth-iterators-and-the-for-of-loop/